core/src/components/modal/modal.scss

```diff
@@ -107,3 +110,35 @@
     --height: #{$modal-inset-height-large};
   }
 }
+
+// Sheet Modal
+// --------------------------------------------------
+
+.modal-handle {
```

Comment:
Thoughts on adding a `grab`/`grabbing` cursor here?

Reply:
The bottom sheet is pretty mobile-focused so I think I would want to see the desktop use case first.